middleware/static/static_test.go

```diff
@@ -738,7 +738,10 @@ func Test_Static_Compress(t *testing.T) {
 			// request non-compressable file (less than 200 bytes), Content Lengh will remain the same
 			req := httptest.NewRequest(fiber.MethodGet, "/css/style.css", nil)
 			req.Header.Set("Accept-Encoding", algo)
-			resp, err := app.Test(req, 10*time.Second)
+			resp, err := app.Test(req, fiber.TestConfig{
+				Timeout:      10 * time.Second,
+				ErrOnTimeout: true,
+			})
```

Comment:
@grivera64 Can you make this change

Reply:
@gaby I've made this change for a few of the files in the middleware that use a single testConfig. A few other middleware used 2 or more different configs, so I didn't use a variable for those to avoid confusion between the configs.